doc/content/3.documentation/1.concepts/testing.md

```diff
@@ -161,6 +161,46 @@ public async Task ASampleTest()
 }
 ```
 
+### Request / Response
+
+To test a Request using the MassTransit Test Harness:
+
+```csharp
+[Test]
+public async Task ASampleTest()
+{
+    await using var provider = new ServiceCollection()
+        .AddMassTransitTestHarness(cfg =>
+        {
+            cfg.Handler<SubmitOrderConsumer>(async cxt => 
+            {
+                await cxt.RespondAsync(new OrderResponse("OK"));
+            }).Endpoint(ep => ep.Name = "ANamedEndpoint");
+        })
+        .BuildServiceProvider(true);
+
+    var harness = provider.GetRequiredService<ITestHarness>();
+
+    await harness.Start();
+
+    // your actual test code may include a call like below some where
+    var clientFactory = provider.GetRequiredService<IScopedClientFactory>();
```

Comment:
`harness.GetRequestClient<OrderSubmitted>()`

Also, using an event name syntax for a request seems broken, vs. `SubmitOrder`.